contracts/src/bridge/SequencerInbox.sol

```diff
@@ -53,6 +53,7 @@ contract SequencerInbox is DelegateCallAware, GasRefundEnabled, ISequencerInbox
 
     IOwnable public rollup;
     mapping(address => bool) public isBatchPoster;
+    mapping(address => bool) public isSequencer;
```

Comment:
Ah, one thing I forgot -- this field will need to go after all the existing ones so we can upgrade into it